controller/organization.go

```diff
@@ -34,7 +34,7 @@ func NewOrganizationController(service *goa.Service, db application.DB) *Organiz
 func (c *OrganizationController) Create(ctx *app.CreateOrganizationContext) error {
 	currentUser, err := login.ContextIdentity(ctx)
 	if err != nil {
-		return jsonapi.JSONErrorResponse(ctx, goa.ErrUnauthorized(err.Error()))
+		return errors.NewUnauthorizedError(err.Error())
```

Comment:
What I implied is `jsonapi.JSONErrorResponse(ctx,errors.NewUnauthorizedError(err.Error()))`

Reply:
I fixed all this, my mistake.